types/time.go

```diff
@@ -292,8 +292,19 @@ const dateFormat = "%Y%m%d"
 // 2012-12-12T10:10:10 -> 20121212101010
 // 2012-12-12T10:10:10.123456 -> 20121212101010.123456
 func (t Time) ToNumber() *MyDecimal {
+	dec := new(MyDecimal)
+	err := t.FillNumber(dec)
+	// We skip checking error here because time formatted string can be parsed certainly.
+	terror.Log(errors.Trace(err))
+	return dec
+}
+
+// FillNumber is the same as ToNumber,
+// but reuses input decimal instead of allocating one.
+func (t Time) FillNumber(dec *MyDecimal) error {
```

Comment:
> We skip checking error here because time formatted string can be parsed certainly.

The comment and error handling should keep in this method and it's not necessary to return an error.

Reply:
Fixed. PTAL